### PR TITLE
prepare 1.12.5

### PR DESCRIPTION
**Pull request by r10s**
bugfix-release, core is the same, but the fixed android4 notifications are probably worth an update.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,14 @@
 # Delta Chat Android Changelog
 
+## v1.12.4
+2020-08
+
+* fix notifications for Android 4
+* fix and streamline querying permissions
+* fix removing POIs from maps
+* make bot-commands as /echo clickable
```
Comment (link2xt) [inline CHANGELOG.md:9]:
```suggestion
* make bot-commands such as /echo clickable
```

**Review by link2xt — APPROVED** [review]

**Comment by r10s** [comment]
thanks for approving! can you have a quick look at https://github.com/deltachat/deltachat-android/pull/1599, i want to get that merged before doing a release (which will be 1.12.5 then)

File: CHANGELOG.md (hunk 1 above)
Comment (r10s) [reply CHANGELOG.md]: thanks, changed that.

**Comment by r10s** [comment]
finally, i noticed some linkify issues, see https://github.com/deltachat/deltachat-android/issues/1600, i will try to fix them (the bugs are currently not released)

**Pull request merged**